.github/workflows/ci.yml

```diff
@@ -104,7 +106,7 @@ jobs:
         fail-fast: false
         matrix:
           #python-version: [3.6]
-          python-version: [3.6, 3.9]
+          python-version: [3.6, 3.9, "3.10"]
```

Comment:
We can drop `3.9` here. Testing for `3.6` and `3.10` should hopefully be enough, since we already have a build test for each supported python version.